Library/Formula/rabbitmq.rb

```diff
@@ -1,14 +1,12 @@
-# We'd really like this formula to be built from source so we can build
-# against a modern OpenSSL rather than the prehistoric system version.
-# Submit a PR if you're interested, Thanks!
```

Comment:
(I did indeed verify that the two audit failures that had been present in the old version of the formula are gone in this branch)